en-US/snippets/2018/jan2018.lang

```diff
@@ -0,0 +1,92 @@
+## NOTE: Reference bug 1429866; Deadline: 1 February, 2018
+## NOTE: Guidelines on localizing snippets: https://goo.gl/PluA97.  The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: Campaign: cross platform promotions on a variety of topics
+## NOTE: Try to limit character length to 160. It is ok if it goes over by a few.
+
+
+## MAX_LENGTH: 160
+# LIMIT: 160 characters. Please rephrase as needed to fit.
+;Sunglasses repel harmful UV rays, make you look cool and give the bridge of your nose something to do. What’s not to love?
+Sunglasses repel harmful UV rays, make you look cool and give the bridge of your nose something to do. What’s not to love?
+
+
+## MAX_LENGTH: 160
+# LIMIT: 160 characters. Please rephrase as needed to fit.
+# Campaign: Mozilla mission
+# To show what Mozilla stands for, if you have a suggestion for a page in your language that the snippet should link to, feel free to include it in your translation.
+;Mozilla is a global non-profit dedicated to putting you in control of your online experience and shaping the future of the web for the public good.
```

Comment:
as a FYI, we could mention (not necessarily here, could be in the email) that a very similar snippet has been translated in July 2017

Reply:
Will include it in the email. Good point.